### PR TITLE
Copy `regression.diffs` after ICG failure [#120858343]

### PR DESCRIPTION
**Pull request by hsyuan**

### CODE REVIEW

File: concourse/scripts/test_with_orca.py [hunk 1]
```diff
@@ -64,6 +68,8 @@ def main():
     if status:
         return status
     status = icg()
+    if status:
+        copy_output()
```
Comment (danielgustafsson) [inline concourse/scripts/test_with_orca.py]: We need to preserve `src/test/regress/regression.out` as well as it contains important information on timing.

Comment (hsyuan) [inline concourse/scripts/test_with_orca.py]: Thanks for reminding. Already fixed it.

**Comment by oarap** [comment]
+1

File: concourse/scripts/test_with_orca.py (hunk 1 above)
Comment (danielgustafsson) [inline concourse/scripts/test_with_orca.py]: Great!

**Pull request merged**